components/gitpod-protocol/src/attribution.ts

```diff
@@ -22,19 +17,15 @@ export interface TeamAttributionId {
 export namespace AttributionId {
     const SEPARATOR = ":";
 
-    export function createFromOrganizationId(organizationId?: string): AttributionId | undefined {
-        return organizationId ? { kind: "team", teamId: organizationId } : undefined;
+    export function createFromOrganizationId(organizationId: string): AttributionId {
+        return { kind: "team", teamId: organizationId };
     }
 
-    export function create(userOrTeam: User | Team): AttributionId {
-        if (User.is(userOrTeam)) {
-            return { kind: "user", userId: userOrTeam.id };
-        } else {
-            return { kind: "team", teamId: userOrTeam.id };
-        }
+    export function create(organization: Organization): AttributionId {
+        return { kind: "team", teamId: organization.id };
```

Comment:
```suggestion
        return AttributionId.createFromOrganizationId(organization.id);
```

Reply:
Is it really true that nowhere in the system we still have UserAttributionId and see value in having common parsing code for it?
I thought we'd go for removing the constructors/helpers, but keep the underlying data type... but maybe we don't access that data anymore/it's gone?

Reply:
Yes, we no longer have user: attribution and if so it is a bug. I have checked the DB and am only aware of the usage table having some credit entries, which I will fix manually.